### PR TITLE
Add Media Library Reference

### PR DESCRIPTION
**Pull request by CommanderFoo**

### CODE REVIEW

File: src/references/media_library.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Content that can be uploaded such as images support JPEG, JPG, and PNG, along with support for transparency.
+
+Creators have a maximum total storage of 104 MB per account.
+
+### Opening the Media Library
+
+The Media Library can be opened by selecting **Media Library** from the **Window** menu. From this window creators can view, rename, and delete content they have uploaded to their account.
+
+Opening the Media Library window in different projects will show that creator's content.
```
Comment (DoubleA-MG) [inline src/references/media_library.md]: So the Media Library is shared across accounts and not stored on the project itself? Might need to reword this sentence to be more clear.

File: src/references/media_library.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+id: media_library_reference
+name: Media Library
+title: Media Library
+tags:
+  - Reference
+---
+
+# Media Library
+
+## Overview
```
Comment (DoubleA-MG) [inline src/references/media_library.md:11]: One thing that took me a second to realize was the difference between the raw image and "brushes". Could use a sentence of explaining the two.

File: src/references/media_library.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+In the image below, the left UI Image does not have **Retain Aspect Ratio** enabled, whereas the right UI image does. This can still be scaled, but will retain the aspect ratio.
+
+![!Ratio](../img/MediaLibrary/ratio.png){: .center loading="lazy" }
+
+#### Brush Picker
+
+The **Brush Picker** window will allow creators to find Core Base Brushes and their own brushes quickly when needing to swap out an existing brush. The Core Base Brushes can be filtered by selecting a type from the filter drop down.
+
+![!Brush Picker](../img/MediaLibrary/brush_picker.png){: .center loading="lazy" }
+
```
Comment (DoubleA-MG) [inline src/references/media_library.md:86]: Did we want to mention Community Content not supporting media at the moment?

**Review by DoubleA-MG — CHANGES_REQUESTED** [review]

File: src/references/media_library.md (hunk 3 above)
Comment (CommanderFoo) [reply src/references/media_library.md:86]: Yeah, going to add that in. Haven't decided where to put it yet.

File: src/references/media_library.md (hunk 2 above)
Comment (CommanderFoo) [reply src/references/media_library.md:11]: Yeah I did read that stuff and think it's a little confusing so kept it simple. Will consider adding a note about it.

**Pull request merged**